src/Altinn.Apps/AppFrontend/react/altinn-app-frontend/src/features/entrypoint/Entrypoint.tsx

```diff
@@ -66,8 +105,27 @@ export default function Entrypoint() {
     return <Instantiate />;
   }
 
+  if (action === 'select-instance' && partyValidation?.valid && activeInstances !== null) {
+    if (activeInstances.length === 0) {
+      // no existing instances exist, we start instantiation
+      return <Instantiate />;
+    }
+    return (
+      // let user decide if continuing on existing or starting new
+      <Presentation
+        header={applicationMetadata?.title?.nb}
+        type={ProcessTaskType.Unknown}
+      >
+        <InstanceSelection
+          instances={activeInstances}
+          onNewInstance={handleNewInstance}
+        />
+      </Presentation>
+    );
+  }
+
   // stateless view
-  if (action && partyValidation?.valid) {
+  if (isStatelessApp(applicationMetadata) && partyValidation?.valid) {
```

Comment:
Flipping the order here could possibly evaluate faster. (Locals versus executing a function).

Reply:
Good catch. Fixed.